lib/tasks/assets.rake

```diff
@@ -6,7 +6,6 @@
 require "active_support"
 
 ENV["RAILS_ENV"] ||= ENV["RACK_ENV"] || "development"
-ENV["NODE_ENV"]  ||= "development"
```

Comment:
I actually think the defaults could be production, as the assets.rake is typically used only for production builds.

That being said, would this be a breaking change, maybe requiring a major version change? While technically yes, I don't think this would hurt any production deployments.

I think most people run assets:precompile with the RAILS_ENV and NODE_ENV explicitly set.

Thoughts?